src/nrtk_explorer/library/dataset.py

```diff
@@ -24,3 +25,20 @@ class Dataset(TypedDict):
     categories: List[DatasetCategory]
     images: List[DatasetImage]
     annotations: List[DatasetAnnotation]
+
+
+def load_dataset(path: str) -> Dataset:
+    with open(path) as f:
+        return json.load(f)
+
+
+dataset_json: Dataset = {"categories": [], "images": [], "annotations": []}
+dataset_path: str = ""
+
+
+def get_dataset(path: str) -> Dataset:
```

Comment:
```suggestion
def get_dataset(path: str, force_reopen = False: bool) -> Dataset:
```

Reply:
Ah. When would we need that?  If dataset files are expected to change, methinks reloading the file with some file watcher callback could be detail of datasets.py.

Reply:
I figured that there might be cases where we need to reload a file, in the filesystem the user my rename or move a dataset from one path to the other. I am envisioning on_dataset_change to invoke this function with this argument.

Reply:
Added the "force" param.

Reply:
Thanks!